content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -187,7 +188,18 @@ replace it with `member4=http://10.0.0.4`.
    fd422379fda50e48, started, member3, http://10.0.0.3:2380, http://10.0.0.3:2379
    ```
 
-2. Remove the failed member:
+2. Do either of the following:
+
+   1. If each Kubernetes API server is configured to communicate with all etcd
+      members, remove the failed member from the `--etcd-servers` flag, then
+      restart each Kubernetes API server.
+   2. If each Kubernetes API server communicates with a single etcd member,
```

Comment:
```suggestion
1. Do either of the following:

   1. If each Kubernetes API server is configured to communicate with all etcd
      members, remove the failed member from the `--etcd-servers` flag, then
      restart each Kubernetes API server.
   1. If each Kubernetes API server communicates with a single etcd member,
```

Reply:
Thanks for fixing the numbering.

Reply:
I like letting markdown being responsible for keeping the numbering accurate, so I'll merge in the use of 1 everywhere like you suggest.